### PR TITLE
Added homepage

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**

### CODE REVIEW

File: src/Pages/Homepage/homepage.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    font-size: 1.7rem;
+}
+
+.link {
+    text-decoration: none;
+    color: black;
+}
+
+.main-container{
+    background-color: #f3f4f6;
```
Comment (shubhamkumar648) [inline src/Pages/Homepage/homepage.css]: suggestion : use color value like u did in component libraray

**Review by shubhamkumar648 — APPROVED** [review]
- remaing l code look good 
- good to merge

File: src/Pages/Homepage/homepage.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    padding: 0;
+    
+}
+
+.main-wrapper{
+    display: flex;
+}
+
+.nav-links {
+    background-color: rgb(212, 54, 54);
```
Comment (ajit-sharmaGH) [inline src/Pages/Homepage/homepage.css]: You can use `css variable`

File: src/Pages/Homepage/homepage.css [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    flex-direction: column;
+}
+
+.sidebar-link-active{
+    color: rgb(212, 54, 54);
+    text-decoration: none;
+}
+
+.hover:hover{
+    background-color: #dadde0;
```
Comment (ajit-sharmaGH) [inline src/Pages/Homepage/homepage.css]: here also

**Review by ajit-sharmaGH — APPROVED** [review]
Suggestion : you can make color variable in `index.css` file under `root { }` 

Everything else is good 👍

**Pull request merged**